lib/generator._js

```diff
@@ -461,6 +454,9 @@ ScriptGenerator.prototype.generateAspNetCoreScript = function (templateFileName,
   argNotNull(templateFileName, 'templateFileName');
 
   var lowerCaseScriptType = this.scriptType.toLowerCase();
+  var fixedRestore = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.restore) : fixPathSeperatorToUnix(options.restore);
```

Comment:
That flag only tells you which platform the script is currently running on, not which OS the *generated* script is supposed to run.
If you can assume that generator.js runs on Kudu where the deploy.ps1 will run, then we can use the flag. Otherwise, we will need to introduce a new option like "-crlf".

Actually I see other generator functions (e.g. generateBasicDeploymentScript at line 419) have the same issue. How about going with it for now as a known issue and provide a proper fix later?